app/schemas/models/user.coffee

```diff
@@ -153,7 +153,8 @@ _.extend UserSchema.properties,
       title: 'Player Ozaria Customization',
       description: 'Player customization options, including hero name, objectId and applied color tints.'
     }, {
-      thangType: c.objectId(links: [{rel: 'db', href: '/db/thang.type/{($)}/version'}], title: 'Thang Type', description: 'The ThangType of the hero.', format: 'thang-type'),
+      # TODO: Temporarily storing the cinematic original id here. Should be the level hero.
+      thangType: c.stringID(links: [{rel: 'db', href: '/db/thang.type/{($)}/version'}], title: 'Thang Type', description: 'The ThangType of the hero.', format: 'thang-type'),
```

Comment:
If this is going to be an id, I think thangTypeId would be a better name, thangType looks a bit confusing.

Reply:
Great comment. I'll use `cinematicThangTypeOriginal` since that is even more specific.